### ISSUE TITLE
last decoration line is not formatted properly with `--wrap never` 

### ISSUE THREAD

**Issue by Rogach** [sharkdp/bat#299]
Example:
```bash
$ echo -n deadbeef > test.txt
$ bat --wrap=never test.txt 
───────┬─────────────────────────────────
       │ File: test.txt
───────┼─────────────────────────────────
   1   │ deadbeef───────┴─────────────────────────────────
```

Compare that with `--wrap=character` (same input file):
```bash
$ bat test.txt
───────┬─────────────────────────────────
       │ File: test.txt
───────┼─────────────────────────────────
   1   │ deadbeef
───────┴─────────────────────────────────
```

**Comment by Rogach** [comment]
I'm using 0.6.1 on Arch Linux:
```
$ bat --version
bat 0.6.1
```

**Comment by Rogach** [comment]
It seems that this section is the culprit:
https://github.com/sharkdp/bat/blob/master/src/printer.rs#L230-L245

It simply prints the line as is, expecting that every line will be ended in a newline. But the last line can end without a newline, resulting in broken formatting.

It is possible to look at the last character and print the missing newline if there isn't one:
```diff
impl<'a> Printer for InteractivePrinter<'a> {
                     )).collect::<Vec<_>>()
                     .join("")
             )?;
+
+            if line.bytes().next_back() != Some(0x0a) {
+                write!(handle, "\n")?;
+            }
         } else {
             for &(style, region) in regions.iter() {
                 let mut ansi_iterator = AnsiCodeIterator::new(region);
```

If you are okay with such solution, I would be happy to create a pull request!

**Comment by Rogach** [comment]
Here you go: #300

**Comment by sharkdp** [comment]
Released in [v0.7.0](https://github.com/sharkdp/bat/releases/tag/v0.7.0).
